tools/ci_build/amd_hipify.py

```diff
@@ -11,9 +11,15 @@
 
 log = get_logger("amd_hipify")
 
-contrib_ops_path = "onnxruntime/contrib_ops"
-providers_path = "onnxruntime/core/providers"
-training_ops_path = "orttraining/orttraining/training_ops"
+
+def path_in_repo(path):
+    repo_root = os.path.relpath(os.path.join(os.path.dirname(__file__), "../.."))
+    return os.path.join(repo_root, path)
```

Comment:
I suggest running a fit command to find the repo root so we can be free to move this file around.

Reply:
The problem with git is that we may not build from a git repo. In some conner cases the user might download a tar ball. Do we have another better alternative? Otherwise, I think we shall stick with this change, even it makes it harder to move the file, but we won't move it in the foreseeable future.

Reply:
Sounds good thanks!